test/nodetool/test_flush.py

```diff
@@ -8,7 +8,13 @@
 import utils
 
 
-def test_flush(nodetool):
+def test_flush_all_tables(nodetool):
```

Comment:
You probably need to add `scylla_only` to this test, as the Java nodetool will not use this new endpoint.

Please make sure to run these test against the Java nodetool as well, and adjust the new tests as needed.
You can do that by e.g.:
```
cd tests/nodetool
pytest --nodetool=cassandra test_flush.py
```

Reply:
done in v5